### PR TITLE
Add `asyncio.base_events.__all__`

### PR DESCRIPTION
**Pull request by AlexWaygood**

### CODE REVIEW

File: stdlib/asyncio/base_events.pyi [hunk 1]
```diff
@@ -14,6 +14,8 @@ from typing_extensions import Literal
 if sys.version_info >= (3, 7):
     from contextvars import Context
 
+__all__ = ["BaseEventLoop"]
```
Comment (JelleZijlstra) [inline stdlib/asyncio/base_events.pyi:17]:
```suggestion
__all__ = ("BaseEventLoop",)
```
stubtest being a pedant

Comment (AlexWaygood) [reply stdlib/asyncio/base_events.pyi]: Thanks!

**Comment by JelleZijlstra** [comment]
I'd be open to making stubtest not care whether `__all__` is a list or a tuple.

**Review by JelleZijlstra — APPROVED** [review]

**Comment by AlexWaygood** [comment]
> I'd be open to making stubtest not care whether `__all__` is a list or a tuple.

More problematic imo is the fact that stubtest doesn't check whether the elements within `__all__` are correct or not. I was trying to find a way today to patch it so that it would, but couldn't figure out how to retrieve that information from the `Var` node mypy generates.

**Pull request merged**